code-scanning/rust-clippy.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
Please do few things in comments at the top:
1. call out the value provided by this starter workflow
2. If possible, link to a "learn more" or "docs" page which may talk more details
3. If there is a pre-requisite/assumption needed to make this workflow succeed on a repo, please call those out.

Reply:
updated.

Reply:
Hmm, I do not see any change in this file..

Reply:
Added comments to yml file.